profile/pom.xml

```diff
@@ -58,6 +58,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
+        <!-- hibernate-java8 is necessary because hibernate-core handles timestamp columns as binary in H2. -->
+        <!-- see: https://stackoverflow.com/questions/44676732/how-to-map-java-time-localdatetime-to-timestamp-in-h2-database-with-hibernate -->
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-java8</artifactId>
```

Comment:
Same, is only needed in scope test?

Reply:
Same:

>In this case it is only necessary for test, but I thought it would be best to use the same libs in production as in our integration tests...